fluentui_core/src/main/res/values/strings.xml

```diff
@@ -1,9 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+    <!-- Describes selected action-->
     <string name="fluentui_selected">Selected</string>
+    <!-- Describes not selected action-->
     <string name="fluentui_not_selected">Not Selected</string>
+    <!-- Describes disabled action-->
     <string name="fluentui_disabled">Disabled</string>
+    <!-- Describes enabled action-->
     <string name="fluentui_enabled">Enabled</string>
+    <!-- Describes close action for a sheet-->
     <string name="fluentui_close_sheet">Close Sheet</string>
+    <!-- Describes close action -->
     <string name="fluentui_close">Close</string>
```

Comment:
Did you rebase it. I think new changes are added which might need comments as well.